File: src/main/java/ru/rsavin/socialnetwork/web/dto/EntityMapper.java
package ru.rsavin.socialnetwork.web.dto;

import ru.rsavin.socialnetwork.domain.Person;

public class EntityMapper {
    public static UserDto fromPerson(Person from) {
        UserDto dto = new UserDto();
        dto.setFirstName(from.getFirstName());
        dto.setSecondName(from.getSecondName());
        dto.setAge(from.getAge());
        dto.setBiography(from.getBiography());
        dto.setCity(from.getCity());
        return dto;
    }
}

File: app/src/main/kotlin/ru/rsavin/socialnetwork/web/dto/EntityMapper.kt
package ru.rsavin.socialnetwork.web.dto

import ru.rsavin.socialnetwork.domain.Person
import ru.rsavin.socialnetwork.domain.Post

object EntityMapper {
    fun fromPerson(from: Person): UserDto = UserDto(
        id = from.id,
        firstName = from.firstName,
        secondName = from.secondName,
        age = from.age,
        biography = from.biography,
        city = from.city
    )

    fun from(from: Post): PostDto = PostDto(
        id = from.id,
        text = from.text,
        authorId = from.authorId
    )
}